plugins/modules/mount.py

```diff
@@ -734,105 +735,116 @@ def main():
     name = module.params['path']
     changed = False
 
-    if state == 'absent':
-        name, changed = unset_mount(module, args)
+    lock_timeout = 2
+    lock_tempdir = '/tmp'
```

Comment:
```suggestion
    lock_tempdir = tempfile.mkdtemp(prefix="ansible_mount")
```

Reply:
the point is to lock `/etc/fstab`, that can't work if each instance of the module creates a lock with a different path, making each process unaware of the locks of the others.

Reply:
an *ansible* tmp path, that we are sure it is shared by all instances of a module within a same task could be fine, but I don't know which/where...

Reply:
I guess we can keep it `/tmp`. Later think about changing since I myself have no experience in Locking.